src/masternodes/balances.h

```diff
@@ -69,15 +70,16 @@ struct CBalances {
         return Res::Ok();
     }
 
-    CBalances SubBalancesWithRemainder(const TAmounts &other) {
+    ResVal<CBalances> SubBalancesWithRemainder(const TAmounts &other) {
         CBalances remainderBalances;
         for (const auto &kv : other) {
-            CTokenAmount remainder = SubWithRemainder(CTokenAmount{kv.first, kv.second});
+            auto resVal = SubWithRemainder(CTokenAmount{kv.first, kv.second});
+            Require(resVal);
```

Comment:
Let's not move away from Require use, and just use simple `if`

Reply:
And in these particular cases, since we have to work with backward compatibility in mind -- why don't we leave a comment behind to short circuit as TODO for later, but for now, let's not break out early. Just use a tracking variable, but still go through all iterations in terms of failure, so this method can be used in a backward compatible way.

Reply:
Resolved in 8c0d1c6dd